src/main/java/frc/robot/subsystems/Feeder.java

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import static frc.robot.Constants.FeederConstants.*;
```

Comment:
Looks like we missed committing the Constants file changes, maybe the Constants file wasn't saved? Need to either locate and commit those changes or remake them.